### PR TITLE
Web3-Onboard - add web3-onboard libs and new wallet hook [2/n]

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1739** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1739" target="_blank">Graphite</a> 
    * **PR #1740** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1740" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1740?utm_source=stack-comment).

File: package.json [hunk 1]
```diff
@@ -33,11 +33,13 @@
     "@uniswap/sdk-core": "3.0.1",
     "@uniswap/v3-sdk": "3.8.2",
     "@walletconnect/web3-provider": "1.8.0",
+    "@web3-onboard/core": "^2.6.0",
```
Comment (benschac) [inline package.json:36]: nit: you could further break this stack up. one PR for the install, and yarn lock, the other for on board.

That way if there's something blocking onboard stuff you could still merge the dep updates on their own.

The rule I use is if and can say `and` it could probably be two different PRs.

This stack looks siiiiiick. Wooooo 🥲🎉

**Review by benschac — COMMENTED** [review]
tiny comment

File: package.json (hunk 1 above)
Comment (wraeth-eth) [reply package.json:36]: Thanks for the heads up @benschac - really liking graphite so far! will do this as soon as I work out how to traverse the stack ;)

Comment (wraeth-eth) [reply package.json:37]: Talked to @benschac and decided not to do this for now as it is feeling too confusing/risky
